src/pages/preferences/preferencesTabs/ProfileTab.tsx

```diff
@@ -96,45 +95,74 @@ export const ProfileTab: React.VFC = () => {
       {/* @ts-expect-error */}
       <PreferencesCard>
         <ContentWrapper>
-          <StyledTextInput
-            label="Github Username"
-            onChange={handleFieldUpdate(setGithubUsernameField)}
-            value={githubUsernameField}
+          <SpruceForm
+            onChange={({ formData, errors }) => {
+              setHasErrors(errors.length > 0);
+              setFormState(formData);
+            }}
+            formData={formState}
+            uiSchema={{
+              timezone: {
+                "ui:placeholder": "Select a timezone",
+              },
+              region: {
+                "ui:placeholder": "Select an AWS region",
+              },
+              githubUser: {
+                lastKnownAs: {
+                  "ui:placeholder": "Enter your GitHub username",
+                },
+              },
+              dateFormat: {
+                "ui:placeholder": "Select a date format",
+              },
+            }}
+            schema={{
+              properties: {
+                githubUser: {
```

Comment:
the graphql mutation expects an object with the shape 
```
githubUser: {
 lastKnownAs: "thing"
}
```

I did this to avoid transforming it later on. I don't think it has any visual change.